distributed/node.py

```diff
@@ -87,13 +87,17 @@ def _setup_logging(self, logger):
         logger.addHandler(self._deque_handler)
         weakref.finalize(self, logger.removeHandler, self._deque_handler)
 
-    def get_logs(self, comm=None, n=None):
+    def get_logs(self, comm=None, n=None, timestamps=False):
```

Comment:
It would be great if you can start off with a doc string for this. Start simple with the changes you contributed. We can add on to it later on.